src/main/java/stringcalculator/domain/Operator.java

```diff
@@ -0,0 +1,38 @@
+package stringcalculator.domain;
+
+public enum Operator {
+    PLUS("+"),
+    MINUS("-"),
+    MULTIPLY("*"),
+    DIVIDE("/");
+
+    private static final String DIVIDE_BY_ZERO_ERROR_MESSAGE = "0으로 나눌 수 없습니다.";
+    private static final String INVALID_OPERATOR_ERROR_MESSAGE = "연산자는 \"+, -, *, %\" 만 가능합니다.";
+    private final String symbol;
+
+    Operator(String symbol) {
+        this.symbol = symbol;
+    }
+
+    public static int operation(String symbol, int precedingOperand, int followingOperand) {
+        if (symbol.equals(PLUS.symbol)) {
+            precedingOperand += followingOperand;
+        } else if (symbol.equals(MINUS.symbol)) {
+            precedingOperand -= followingOperand;
+        } else if (symbol.equals(MULTIPLY.symbol)) {
+            precedingOperand *= followingOperand;
+        } else if (symbol.equals(DIVIDE.symbol)) {
+            divideValidate(followingOperand);
+            precedingOperand /= followingOperand;
+        } else {
+            throw new IllegalArgumentException(INVALID_OPERATOR_ERROR_MESSAGE);
+        }
```

Comment:
```suggestion
    public static int operation(String symbol, int precedingOperand, int followingOperand) {
        if (symbol.equals(PLUS.symbol)) {
            return precedingOperand + followingOperand;
        }
        if (symbol.equals(MINUS.symbol)) {
            return precedingOperand - followingOperand;
        }
        if (symbol.equals(MULTIPLY.symbol)) {
            return precedingOperand * followingOperand;
        }
        if (symbol.equals(DIVIDE.symbol)) {
            divideValidate(followingOperand);
            return precedingOperand / followingOperand;
        }
        throw new IllegalArgumentException(INVALID_OPERATOR_ERROR_MESSAGE);
    }
```

> 해당 미션에서 else 를 사용하지 않고 계산을 처리하는 방법을 고민해봤는데, 마땅한 방법이 떠오르지 않습니다ㅠㅠ...작은 조언이라도 부탁드리겠습니다.

이런 방식은 어떠신가요?
조금 더 함수형스러운 방식이 있기는 한데 (enum 을 key 로 해서 처리해야할 연산자체를 value 로 하는 Map 자료 구조 만들기)
자바는 사실 전혀 함수형스럽지 않은 언어라서,
저는 자바의 경우 무리하게 함수형으로 짜는 것을 선호하지 않는 스타일입니다.